src/hosts/buttonservicehost.ts

```diff
@@ -1,13 +1,11 @@
 import { ButtonEvent, REFRESH, SRV_BUTTON } from "../jdom/constants";
 import SensorServiceHost from "./sensorservicehost";
 
-const LONG_CLICK_DELAY = 500
-const HOLD_DELAY = 1500
+const CLICK_HOLD_TIME = 500
```

Comment:
We should support constants in the spec

Reply:
agreed. You already filed an issue on that. Also, click_hold_time is a rw register - see button.md. I just didn't implement the logic for it in the host.